app/javascript/packages/document-capture/components/address-search.spec.tsx

```diff
@@ -20,7 +20,7 @@ const DEFAULT_RESPONSE = [
   },
 ];
 
-describe('AddressSearch', () => {
+describe('AddressSearch finds an address', () => {
```

Comment:
Nit: I might expect "finds an address" and "does not find an address" to be contexts, both of which describing the behavior of `AddressSearch`.

```ts
describe('AddressSearch', () => {
  context('when an address is found', () => {
    // ...
  });

  context('when an address is not found', () => {
    // ...
  });
});
```

Reply:
Good idea, thanks. 👍🏻